index.html

```diff
@@ -719,10 +719,10 @@ <h4>Clave</h4>
 					</section>
 
 
-				<!-- Program -->
+				<!-- Agenda -->
 					<section id="program" class="wrapper style6 fade-up">
```

Comment:
Nit: maybe you want to change the anchor name here and everywhere.

Reply:
Changed